.github/workflows/ci-documentbot.yml

```diff
@@ -43,4 +43,50 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           LABEL_WATCH_LIST: 'doc,doc-required,doc-not-needed,doc-complete'
-          LABEL_MISSING: 'doc-label-missing'
\ No newline at end of file
+          LABEL_MISSING: 'doc-label-missing'
+
+  semantic:
+    name: Check PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v4
+        continue-on-error: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            feat
+            improve
+            fix
+            cleanup
+            refactor
+            revert
+
+          scopes: |
+            admin
+            broker
+            cli
+            io
+            fn
+            meta
+            monitor
+            proxy
+            schema
+            sec
+            sql
+            storage
+            offload
+            txn
+            java
+            cpp
+            py
+            ws
+            test
+            ci
+            build
+            misc
+            doc
+            blog
+            site
+
+          headerPattern: '^(?:\[(\w+)\])?(?:\[(\w+)\])? (.+)$'
```

Comment:
This job should be in a new workflow. We don't run it for every labeled/unlabeled event.

Reply:
We can filter out label events with an if condition

Reply:
But why? They are for totally different things.

Reply:
I know that community want not to add a new workflow, there are many workflow need to management, and the other task is add labels based on pr title, so I want to add it as a job in this workflow.

Reply:
Since @lhotari & @nicoloboschi are working on this domain now. I'd like to pull them for inputs on this topic.

Reply:
I think it's good to consolidate docs-related jobs into one workflow.

Reply:
@maxsxu checking PR title is among the whole project and actually a project basic settings. It's no way to be a docs-related only job.

Reply:
But if you insist on this topic, I don't think it's a blocker. Pay as you go.

Reply:
Good catch! We should add a new workflow, which only listens to the `synchronize` event on PR, and then you also add `subjectPatternError` value on `amannn/action-semantic-pull-request@v4`.